### ISSUE TITLE
Enhancement: ValueList - Add ability to drag and drop with keyboard

### ISSUE THREAD

**Issue by asangma** [Esri/calcite-app-components#356]
### Summary
User should be able to tab into the drag handle, hit space, then use the up/down arrows to move items.

### Actual Behavior
Can't tab into drag handle.

### Expected Behavior
Be able to tab into drag handle.

### Reproduction Steps

1. demo page

### Relevant Info 
See how Bookmarks widget works in edit mode.

cc @driskull @kat10140

**Comment by pr3tori4n** [comment]
@driskull did you use sortable for the Bookmarks widget? If so can I scavenge the code you used to get that working - can you share a code snippet?

**Comment by driskull** [comment]
Yes, its in the bookmarks view.

**Comment by pr3tori4n** [comment]
Modifying this to be an enhancement issue, since its a request for additional functionality that wasn't present on first pass.

**Comment by pr3tori4n** [comment]
@kat10140 please verify. Please double-check Edge and Safari.

**Comment by kat10140** [comment]
works in Chrome, Firefox, Safari, Edge. Closing
